samples/commands-and-operations/app/layout.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<layout xmlns="https://geocortex.com/layout/v1" xmlns:custom="your.custom.namespace">
```

Comment:
The custom namespace isn't really needed in this example.

Reply:
Good call. Removed.